docs/installation.rst

```diff
@@ -0,0 +1,55 @@
+.. _installation:
+
+************
+Installation
+************
+
+Installing ``hvpy``
+-------------------
+
+If you are new to Python and/or do not have familiarity with `Python virtual environments <https://docs.python.org/3/tutorial/venv.html>`__, then we recommend starting by installing `miniforge <https://github.com/conda-forge/miniforge#miniforge3>`__.
+This works on all platforms (Linux, Mac, Windows) and installs a full-featured Python environment in your user directory without requiring root privileges.
+
+Using pip
+---------
+
+.. warning::
+
+    Users of any conda distribution should follow the instructions for :ref:`conda_install`.
+
+To install ``hvpy`` with `pip <https://pip.pypa.io/en/stable/>`__, run ::
+
+    pip install hvpy
+
+.. _conda_install:
+
+Using Conda
+-----------
+
+To install ``hvpy`` using `conda <https://docs.conda.io/projects/conda/en/latest/>`__ run ::
```

Comment:
This brings up the question on whether or not we'll be supporting Anaconda, I haven't worked with it much myself, I think this is worth bringing up on Thursday with Nabil and Jack.

Reply:
We should be submitting a recipe to conda-forge when we have a release of 0.1 out.